hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1032,6 +1033,37 @@ public String getWriteMetaKeyPrefixes() {
     return props.getProperty(WRITE_META_KEY_PREFIXES_PROP);
   }
 
+  /**
+   * Error table configs.
+   */
+  public boolean enableErrorTable() {
```

Comment:
enableErrorTable-> errorTableEnabled ?  this will be more unify

Reply:
ok